src/index.html

```diff
@@ -5,8 +5,288 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="container">
+        <div class="logo">
+          <a class="logo__link" href="#home">
+            <img class="logo__image"
+            src="./images/logo.png" alt="logo">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="is-active">
```

Comment:
I would create a class for this item, to avoid styling by tag. You want your styles to be the same for each item, no matter which tag is used.
```suggestion
              <a href="#" class="nav__link is-active">
```